sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-retraining-pipeline.ipynb

```diff
@@ -0,0 +1,440 @@
+    "                optimizer=Choice([\"sgd\", \"adam\", \"adamw\"]),\n",
+    "                min_size=Choice([600, 800]),  # model-specific\n",
+    "            ),\n",
+    "        ]\n",
+    "    )\n",
+    "    \n",
+    "    image_object_detection_node.set_image_model(early_stopping=True, evaluation_frequency=1)\n",
+    "    image_object_detection_node.set_sweep(\n",
+    "        max_trials=10,\n",
+    "        max_concurrent_trials=2,\n",
+    "        sampling_algorithm=\"Random\",\n",
+    "        early_termination=BanditPolicy(evaluation_interval=2, slack_factor=0.2, delay_evaluation=6),\n",
+    "    )\n",
+    "    \n",
+    "    # provide model name to register the model\n",
+    "    model_register_job = register_model(\n",
+    "        automl_best_run=image_object_detection_node.outputs.best_model,\n",
+    "        primary_metric= image_object_detection_node.primary_metric.value,\n",
+    "        training_mltable = image_object_detection_train_data,\n",
+    "        model_name = \"automl_imagde_object_detection_model\",\n",
+    "        register_always = False\n",
+    "    )\n",
+    "\n",
+    "\n",
+    "data_folder = \"./data\"\n",
+    "pipeline = automl_image_object_detection_retraining(\n",
+    "    image_object_detection_train_data=Input(path=f\"{data_folder}/training-mltable-folder/\", type=\"mltable\"),\n",
+    "    image_object_detection_validation_data=Input(path=f\"{data_folder}/validation-mltable-folder/\", type=\"mltable\"),\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2.2 Submit pipeline job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# submit the pipeline job\n",
+    "pipeline_job = ml_client.jobs.create_or_update(\n",
+    "    pipeline, experiment_name=\"retraining_pipeline\"\n",
+    ")\n",
+    "pipeline_job"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wait until the job completes\n",
+    "ml_client.jobs.stream(pipeline_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Next Steps\n",
+    "You can see further examples of running a pipeline job [here](../)"
+   ]
+  }
+ ],
+ "metadata": {
+  "description": {
+   "description": "Create pipeline with automl node"
+  },
+  "interpreter": {
+   "hash": "36d30df7a6d3e24beb83d3caf7447873d4ac8f8ef4c84dc8e7bbc05068236de9"
+  },
+  "kernelspec": {
+   "display_name": "Python 3 (ipykernel)",
+   "language": "python",
+   "name": "python3"
+  },
+  "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.7"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 4
+}
```

Comment:
[General] I tried this notebook locally and the run fails at the register step. https://ml.azure.com/runs/sweet_double_gjfz0p4bm1?wsid=/subscriptions/dbd697c3-ef40-488f-83e6-5ad4dfb78f9b/resourcegroups/nilesh-new-rg/workspaces/nilesh-ws-new&tid=72f988bf-86f1-41af-91ab-2d7cd011db47&isGlobalView=false#

Seems like some issue while building the environment. Could you please schedule a run and see if you can reproduce the same error? Thanks